README.md

```diff
@@ -7,3 +7,35 @@ slog is a library for capturing structured log information. In contrast to "trad
 * captures arbitrary key-value metadata on each log event
 
 slog forwards messages to [`log`](https://golang.org/pkg/log/) by default. But you probably want to write a a custom output to make use of the context and metadata. At [Monzo](https://monzo.com/), slog captures events both on a per-service and a per-request basis (using the context information) and sends them to a centralised logging system. This lets us view all the logs for a given request across all the micro-services it touches.
+
+## Usage
+
+Internally at Monzo, we recommend that users always prefer structured logging where possible. An example of using slog for this would be:
+
+```go
+slog.Info(ctx, "Loading widget", map[string]interface{}{
+    "stage": "reticulating splines",
+})
+```
+
+When logging an error, we recommend attaching the actual error in the "error" key:
+
+```go
+err := errors.New("No splines to reticulate")
+slog.Error(ctx, "Failed to load widget", map[string]interface{}{
```

Comment:
I completely agree with the idea of passing the error into `slog` directly, but doing it as part of the metadata map with a "magic" key feels a bit cumbersome to me. I wonder whether it would be worth introducing a method just for slogging an error, that takes an `error` as a typed parameter, like:

```go
slog.Error(ctx, err, …)
```

The challenge is what to call it, given `Error` is already taken 🤔 (unless we renamed the existing method).

Reply:
I actually had this behaviour already, but removed it to make this PR a little bit more focussed on the metadata rather than the conventions around keys within it. I think the point is valid, but maybe we can move that discussion to the next PR which has specifics for errors.

Reply:
That's fine. A bit nit-picky and I wouldn't block merging on this, but maybe adding this to the README should be part of the subsequent PR in that case.